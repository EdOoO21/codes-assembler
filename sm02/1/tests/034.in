-269 199 -944 727 -531 840 -939 806 -204 -254 954 943 -868 -132 956 -373 -302 -807 196 47 684 -683 -803 -439 -290 250 -997 590 -683 541 72 -824 640 -875 -244 -478 515 130 443 717 -878 639 -208 912 -659 205 789 303 -366 -403 -219 -358 -565 -579 -73 96 -633 243 -732 173 471 -198 113 -132 -517 -492 400
