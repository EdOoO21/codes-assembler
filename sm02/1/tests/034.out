400
-492
-517
-132
113
-198
471
173
-732
243
-633
96
-73
-579
-565
-358
-219
-403
-366
303
789
205
-659
912
-208
639
-878
717
443
130
515
-478
-244
-875
640
-824
72
541
-683
590
-997
250
-290
-439
-803
-683
684
47
196
-807
-302
-373
956
-132
-868
943
954
-254
-204
806
-939
840
-531
727
-944
199
-269
